src/lepton/stream_interfaces.hh

```diff
@@ -60,7 +60,7 @@ struct ResizableByteBufferList {
 
 class VP8ComponentDecoder_SendToVirtualThread {
     ResizableByteBufferList vbuffers[Sirikata::MuxReader::MAX_STREAM_ID];
-    // Auto-release all buffers allocated in this class by saving them into:
+    // Save here all buffers allocated in this class in order to delete them once the entire file was processed:
```

Comment:
This change should be in previous commit

Reply:
There were two commits - first one released all buffers automatically when owning object is destroyed. I found that this approach doesn't work because the decoder object was released only once when all files are processed, and added second commit, releasing buffers at "safe points" by explicitly calling free_buffers() between images. The comment was changed according to that.

Finally, I modified code to [release the decoder](https://github.com/Bulat-Ziganshin/lepton/commit/abbc90bb5f16673ef39b0eda853f194ae4ce4b19) after each file. So, strictly saying the second commit is overkill, but I want to keep you since we may switch back to retaining the decoder object in future.